package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "infestines",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Note that 0.3.0 hasn't yet been released.  I'll try to finish 0.3.0 this weekend.